### PR TITLE
Add aqua cli documentation

### PR DESCRIPTION
**Pull request by VipulMascarenhas**
This PR adds documentation along with examples on how users can access Aqua resources via ads CLI.

### CODE REVIEW

File: ai-quick-actions/cli.md [hunk 1]
```diff
@@ -0,0 +1,821 @@
...
+
+
+
+# Model Evaluation
+
+## Create Model Evaluation
+
+### Description
+
+Creates a new evaluation model using an existing Aqua model or model deployment. Currently, evaluation is only supported via model deployment as the source.
```
Comment (mrDzurb) [inline ai-quick-actions/cli.md:396]: I think we should remove any mention of the supporting evaluating offline models. Currently we only support deployed models.

File: ai-quick-actions/cli.md [hunk 2]
```diff
@@ -0,0 +1,821 @@
...
+
+```bash
+ads aqua evaluation create [OPTIONS]
+```
+
+### Required Parameters
+
+`--evaluation_source_id [str]`
+
+The evaluation source id. Must be either model or model deployment ocid.
```
Comment (mrDzurb) [inline ai-quick-actions/cli.md:408]: only model deployment

File: ai-quick-actions/cli.md [hunk 3]
```diff
@@ -0,0 +1,821 @@
...
+
+`--evaluation_name [str]`
+
+The name for evaluation.
+
+`--dataset_path [str]`
+
+The dataset path for the evaluation. Could be either a local path from notebook session or an object storage path.
```
Comment (mrDzurb) [inline ai-quick-actions/cli.md:416]: Let's not mention the local path maybe at all in the CLI?

File: ai-quick-actions/cli.md [hunk 4]
```diff
@@ -0,0 +1,821 @@
...
+
+`--project_id [str]`
+
+The project OCID where evaluation is to be created. If not provided, then it defaults to user's project.
+
+`--evaluation_description [str]`
+
+The description of the evaluation. Defaults to None.
+
+`--memory_in_gbs [float]`
```
Comment (mrDzurb) [inline ai-quick-actions/cli.md:453]: This is only for the flexible shapes. Despite we only support flexible shapes for now, in future we will return the other shapes as well.

File: ai-quick-actions/cli.md [hunk 5]
```diff
@@ -0,0 +1,821 @@
...
+The log group id for the evaluation job infrastructure. Defaults to None.
+
+`--log_id [str]`
+
+The log id for the evaluation job infrastructure. Defaults to None.
+
+
+### Example
+
+```bash
```
Comment (mrDzurb) [inline ai-quick-actions/cli.md:484]: We also added supporting the ```rouge``` metric. 

```
--metrics '[{"name": "bertscore", "args": {}}, {"name": "rouge", "args": {}}]
```

File: ai-quick-actions/cli.md [hunk 6]
```diff
@@ -0,0 +1,821 @@
...
+    "parameters": {
+        "max_tokens": 500,
+        "top_p": 1,
+        "top_k": 50,
+        "temperature": 0.7,
+        "presence_penalty": 0,
+        "frequency_penalty": 0,
+        "stop": [],
+        "shape": "VM.Standard.E3.Flex",
+        "dataset_path": "oci://aqua-eval-test-bucket@namespace/dataset/evaluation-sample-with-sys-message.jsonl",
```
Comment (mrDzurb) [inline ai-quick-actions/cli.md:581]: I guess the real path should be removed?

File: ai-quick-actions/cli.md [hunk 7]
```diff
@@ -0,0 +1,821 @@
...
+
+The name for evaluation.
+
+`--dataset_path [str]`
+
+The dataset path for the evaluation. Could be either a local path from notebook session or an object storage path.
+
+`--report_path [str]`
+
+The report path for the evaluation. Must be an object storage path.
```
Comment (mrDzurb) [inline ai-quick-actions/cli.md:420]: Would it make sense to add supported values for every lag just below the description?

```
--report_path [str]
```

The evaluation's dataset path, which must be an Object Storage path.
Example: ```oci://<bucket>@<namespace>/path/to/the/dataset.jsonl```

File: ai-quick-actions/cli.md (hunk 1 above)
Comment (VipulMascarenhas) [reply ai-quick-actions/cli.md]: changed to `Creates a new evaluation model using an existing Aqua model deployment.`

File: ai-quick-actions/cli.md (hunk 2 above)
Comment (VipulMascarenhas) [reply ai-quick-actions/cli.md]: updated.

File: ai-quick-actions/cli.md (hunk 5 above)
Comment (VipulMascarenhas) [reply ai-quick-actions/cli.md:484]: added

File: ai-quick-actions/cli.md (hunk 6 above)
Comment (VipulMascarenhas) [reply ai-quick-actions/cli.md]: yea this is a made-up path as well, but I updated it to `oci://aqua-eval-test-bucket@namespace/dataset/evaluation-sample.jsonl`

File: ai-quick-actions/cli.md (hunk 7 above)
Comment (VipulMascarenhas) [reply ai-quick-actions/cli.md:420]: make sense, I'll some details in the description section for each param.

**Review by darenr — APPROVED** [review]

**Pull request merged**